src/app/zap-templates/zcl/data-model/chip/keypad-input-cluster.xml

```diff
@@ -27,12 +27,12 @@ limitations under the License.
 
     <command source="client" code="0x00" name="SendKey" response="SendKeyResponse" optional="false">
       <description>Upon receipt, this SHALL process a keycode as input to the media device.</description>
-      <arg name="keyCode" type="CecKeyCode"/>
+      <arg name="KeyCode" type="CecKeyCode"/>
```

Comment:
My copy of the spec has the type just as `uint8`.

Reply:
Yes, but the authors of this cluster and the code that uses it insisted on defining this enum (which is not in the spec, because it's in a different spec)....

We can try to address that, but it involves a bunch of code changes and convincing people, so I ignored it for now.

Reply:
Note that my notes on this cluster, which will turn into follow-up issues, include:
```
  - Has a CecKeyCode enum that is not in the spec.
```

Reply:
Should we change it in the spec right away also?

Reply:
We can't easily change it in the spec last I checked.  Hence it being a follow-up issue...